### ISSUE TITLE
generated ydk-py unusable if device advertises deviations not compiled into ydk-py

### ISSUE THREAD

**Issue by einarnn** [CiscoDevNet/ydk-gen#379]
This code:

```
    @classmethod
    def get_active_deviation_tables(cls, capabilities, entity):
        """ Return active deviation tables

            Args:
                cls: First argument for class method
                capabilities: List of capabilities get from provider
                enttiy: An instance of YDk object

            Returns:
                deviation_tables: A dictionary for deviation tables

        """
        active_pmodule_names = _get_active_deviation_module_names(capabilities, entity)
        deviation_tables = {}
        for pname in active_pmodule_names:
            module = importlib.import_module('ydk.models._deviate._%s' % pname)
            deviation_table = getattr(module, '_deviation_table')
            deviation_tables[pname] = deviation_table

        return deviation_tables
```

...generates an exception if a device advertises a deviation module that isn't already compiled into the ydk-py modules being used. This makes a "vanilla" ydk-py unusable if a device advertises deviations not already known. For example, with the IETF bundle currently in the profiles directory and a CSR1000v running IOS-XE 16.5.1 (fails because `ietf-ip-devs` is not available).

Can be worked around by commenting out the for loop in `get_active_deviation_tables`.

**Comment by 111pontes** [comment]
The current ydk-py base doesn't support that kind of forward compatibility.  The XE device should have the same or earlier version of the XE bundle.  Once ydk-py uses the new infra, deviations should be completely handled dynamically based on what the device advertises as capabilities and what it returns as deviation schema.

**Comment by einarnn** [comment]
Yes, I talked to Abhi and understand that this will be resolved in a subsequent release with the new infra. But this is an issue today, and I wouldn't really call this forward-compatibility.

Since we have different deviations across different IOS-XE platforms (i.e. routers & switches differ slightly) it is not currently possible for a developer to use a single ydk-py for, for example "IOS-XE 16.5.1". And note that we may have the same situation on IOS-XR platforms. Can't speak for other vendors' platforms, of course, but this means that with deviation support enabled as currently implemented, we have a problem. Hence the workaround above that I have tested that allows a developer to use a single ydk-py across platforms that differ in terms of deviations.
